pkgs/development/compilers/blueprint/default.nix

```diff
@@ -51,6 +81,5 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "https://gitlab.gnome.org/jwestman/blueprint-compiler";
     license = licenses.lgpl3Plus;
     maintainers = with maintainers; [ benediktbroich ranfdev ];
-    platforms = platforms.unix;
```

Comment:
This should be kept.

Reply:
I wanted to test if it builds on platforms.all.

Reply:
Omitting it does not mean `platforms.all`, though. And that is really impossible to test anyway.